tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -109,6 +109,70 @@ def load_vulnerability_detector_configurations(host_manager):
 
     return configurations
 
+@pytest.fixture(scope='module')
+def install_npm(host_manager: HostManager):
+    """Ensure npm is installed on macOS agents."""
+    for host in host_manager.get_group_hosts('agent'):
+        os_type = host_manager.get_host_variables(host).get('os')
+
+        if os_type.startswith('macos'):
```

Comment:
Consider defining a variable for a list of OS for which we are installing npm and make reference to that in this fixture

In addition, make use of the group `macos` instead of `agent` in order to avoid further checks

Reply:
Modified here: https://github.com/wazuh/wazuh-qa/pull/5158/commits/abb850f24b579bb0561da12d822b0ec10e01d7d6 and check in [this comment.](https://github.com/wazuh/wazuh-qa/issues/5128#issuecomment-2022576419)